src/Compatibility/Microsoft.DotNet.ApiCompatibility/DiagnosticIds.cs

```diff
@@ -17,6 +17,7 @@ public static class DiagnosticIds
         public const string CannotRemoveBaseType = "CP0007";
         public const string CannotRemoveBaseInterface = "CP0008";
         public const string CannotSealType = "CP0009";
+        public const string EnumValuesMustMatch = "CP0010";
```

Comment:
You can do this after the PR goes in, but eventually you'll have to add documentation for this diagnostic as well as how to address it. This is where it should go: [https://docs.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids](https://docs.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids)

Reply:
I'm guessing these are in the dotnet/docs repo?